test/fixtures/BootstrapCommand/extra/lerna.json

```diff
@@ -0,0 +1,8 @@
+{
+  "lerna": "__TEST_VERSION__",
+  "version": "1.0.0",
+  "packages": [
+    {"glob": "packages/*/package.json"},
+    {"glob": "package-3/package.json"}
```

Comment:
@bclinkinbeard, I should have clarified: #392 doesn't solve the Appveyor permissions problem, just the path on that one test…